cmd/ocsp-updater/main.go

```diff
@@ -586,14 +588,32 @@ func setupClients(c cmd.OCSPUpdaterConfig, stats metrics.Scope) (
 	core.StorageAuthority,
 ) {
 	amqpConf := c.AMQP
-	cac, err := rpc.NewCertificateAuthorityClient(clientName, amqpConf, stats)
-	cmd.FailOnError(err, "Unable to create CA client")
+
+	var cac core.CertificateAuthority
+	if c.SAService != nil {
```

Comment:
I think this should be `c.CAService`, not `c.SAService`.

Did the CA gRPC client change get missed when you gRPC-ified the CA?